src/views/tutorial-view/server.ts

```diff
@@ -20,25 +20,25 @@ import { getTutorialsBreadcrumb } from './utils/get-tutorials-breadcrumb'
 
 export interface TutorialPageProps {
   tutorial: TutorialData
-  product: TutorialPageProduct // controls the ProductSwitcher
+  product: ProductData
   layoutProps: TutorialSidebarSidecarProps
   nextCollection?: ClientCollectionLite | null // if null, it is the last collection in the sidebar order
 }
 
 /**
- *  This slug needs to use the Learn product option enum,
- *  as the types in `ProductContext` slug aren't valid for the API req
+ * Given a ProductData object (imported from src/data JSON files) and a tutorial
+ * slug, fetches and returns the page props for
+ * `/{productSlug}/tutorials/{collectionSlug}/{tutorialSlug}` pages.
+ *
+ * Returns the given ProductData object unmodified as the `product` page prop,
+ * which is needed for other areas of the app to function.
```

Comment:
Will let this comment do the initial explaining, and absolutely happy to answer any questions. 💖